package.json

```diff
@@ -110,9 +110,10 @@
     "jssha": "^2.3.1",
     "jwt-decode": "^2.2.0",
     "libxmljs2": "^0.21.3",
+    "lodash": "^4.17.10",
     "marsdb": "^0.6.11",
     "morgan": "^1.9.1",
-    "moment": "^2.3.0",
+    "moment": "^2.1.0",
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `moment` violates your company's policies.

Lifecycle recommends using version [2.19.3](https://www.npmjs.com/package/moment/v/2.19.3) instead as it does not violate any policies.